backend/migration.ts

```diff
@@ -62,7 +73,8 @@ function runMigration(loggerFactory: LoggerFactory, dbModule: DbModule): void {
                         }
                     ];
                     return {
-                        name,
+                        // NOTE: Always changing extension to .js to maintain backward compatibility
+                        name: changeExtensionToJs(name),
```

Comment:
Since migrations are internal part of the application I can't think of any scenario where such backward compatibility could be applicable.

Reply:
I'll explain a bit the reasoning. When old snapshots (e.g. from version 6.3) will be uploaded to fresh installation of 6.4 version, then in `SequelizeMeta` table (the table used by `umzug`) there will be a list of filenames ending with `.js` extension, but in the file system we will have files ending with `.ts` extension. That will lead to the situation where `umzug` is detecting all migrations as pending (and not executed already) and that's something we want to avoid, I think. 

And I wanted to avoid changing `SequelizeMeta` table, so decided to use that kind of approach. 

What do you think?

Reply:
Ah, OK, I didn't think of it 🙂. So it looks like it's actually crucial to have it.

Reply:
Maybe worth to provide a little bit more detailed comment explaining the need for this? It's not something obvious.

Reply:
Sure, make sense. I'll improve this comment.

Reply:
Done.